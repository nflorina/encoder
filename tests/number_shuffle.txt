-181
-141913
-1211311
111142
-1612634
129591
13843
14897
192
178
-16618415
164111
168
16921132
15371791
-14441271
1128
1123487
-141211
17116419
-1386
-17671982
23
END
