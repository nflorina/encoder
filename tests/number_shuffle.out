0 0 22
118131419111211342111112634169591128431397148921817151661846411118169211321691153717114441278112871123411141291711641138617671982
971481121181692411116184611321341693421169115234111711641138617671982111269218114441715161181327811141912871114129371715911284313
